plugin-dev/Source/Sentry/Private/GenericPlatform/GenericPlatformSentrySubsystem.cpp

```diff
@@ -228,6 +228,12 @@ void FGenericPlatformSentrySubsystem::InitWithSettings(const USentrySettings* se
 		: *settings->GetFormattedReleaseName()));
 
 	sentry_options_set_dsn(options, TCHAR_TO_ANSI(*settings->Dsn));
+#if WITH_EDITOR
+	if (!settings->EditorDsn.IsEmpty())
```

Comment:
We should add this to [AppleSentrySubsystem init](https://github.com/getsentry/sentry-unreal/blob/4fa31b7f2ca4d4c2282cb1f4d6cac24f04da1478/plugin-dev/Source/Sentry/Private/Apple/AppleSentrySubsystem.cpp#L41) so Mac users can also benefit from separating editor and packaged game events.